tests/test_validator_integration.py

```
@@ -0,0 +1,970 @@
+                "model_name": "microsoft/deberta-v3-small",
+                "span_mode": "markerV0",
+                "max_len": 384,
+            }
+        }
+        vr = validate_config(gliner_format)
+        assert len(vr.errors) > 0, (
+            "training_cli should reject the 'gliner_config:' format"
+        )
+
+        # The incompatibility is proven: neither format works for both
+        assert not config_ok_with_model_format, (
```

Ruff (0.15.1)

[error] Undefined name `config_ok_with_model_format`

(F821)